plugins/AudioFileProcessor/AudioFileProcessorWaveView.h

```diff
@@ -171,6 +171,7 @@ public slots:
 	void updateGraph();
 	void reverse();
 	void updateCursor(QMouseEvent* me = nullptr);
+	bool pointerCloseToStartEndOrLoop(QMouseEvent* me);
```

Comment:
The parameter and the method could be made `const` (sorry, I did not have that in my proposal):
```
bool pointerCloseToStartEndOrLoop(const QMouseEvent* me) const;
```

Assuming that `position` is also a `const` (or static) method that it takes a `const QMouseEvent` (I don't have the Qt6 branch checked out).

Reply:
position does not take a const, but I can make it. I'm not sure what implications that'll have though (again, not mainly a C++ dev) so if it's problematic, please tell me 😅

Reply:
The changes in `DeprecationHelper` are fine. Otherwise you would have noticed by things not compiling anymore. 😅

Can you please also make the method `pointerCloseToStartEndOrLoop` itself `const`? It would need to be changed as follows in the header (mind the `const` at the end of the line):
```
bool pointerCloseToStartEndOrLoop(const QMouseEvent* me) const;
```

And in the CPP:
```
bool AudioFileProcessorWaveView::pointerCloseToStartEndOrLoop(const QMouseEvent * me) const
```

Reply:
oops, didn't see that

Reply:
No problem! Thanks for the final fix-up!